src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/__tests__/FeaturedCollectionsRails.test.tsx

```diff
@@ -53,22 +54,18 @@ describe("FeaturedCollectionsRails", () => {
     expect(component.text()).toMatch("Street Art: Superheroes and Villains")
   })
 
-  it("Renders arrows when there are more than 4 featured collections", () => {
-    props.collectionGroup.members = [
-      memberData(),
-      memberData(),
-      memberData(),
-      memberData(),
-      memberData(),
-    ]
+  // it("Renders correct height at smaller breakpoints", () => {
+  //   const component = mount(
+  //     <MockBoot breakpoint="sm">
+  //       <FeaturedCollectionsRails {...props} />
+  //     </MockBoot>
+  //   )
 
-    const component = mount(
-      <MockBoot>
-        <FeaturedCollectionsRails {...props} />
-      </MockBoot>
-    )
-    expect(component.find(ArrowButton).length).toBe(2)
-  })
+  //   console.log("comp", component.props())
+  //   console.log("key", component.key())
+
+  //   expect(renderCarousel).toBeCalledWith("430px")
+  // })
```

Comment:
This test is just about there. I need to properly mock `renderCarousel` and update the assertion to include the members prop and tracking function that gets passed to renderCarousel.

I forgot how to mock renderCarousel so that it's in scope with the render/mount of the component on line 58. Any ideas @pepopowitz?

Reply:
Is `jest.spyOn` what you're looking for? https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname

Reply:
Also, for the record, I don't feel like it's a requirement that this test exists, if it's too much of a challenge to get working.